en/components/grids_templates/row_selection.md

```diff
@@ -258,9 +258,240 @@ public getSelectedRows(args) {
 }
 ```
 
-### Styling Guidelines TODO
+### Row selectors templating
+You can template header and row selectors in the `@@igSelector` and also access their contexts which provide useful functionality for different scenarios.
+
+#### igxRowSelector
+With the `igxRowSelector` directive you can bind to the `@@igSelector` row context and access the following members:
```

Comment:
Um, the directive has no such functionality!
You define the custom template by declaring a `<ng-template>` within the grid with an `igxRowSelector` directive applied. Then once the template has been defined, the grid provides the context to it with an _implicit_ context value, that allows you to access such and such properties.

Reply:
Also do describe the grid handles interactions for selection higher than the template, so it's function is intended to be mostly representation.